sharepoint/sharepoint-server-ps/sharepoint-server/Set-SPWebApplication.md

```diff
@@ -337,14 +437,17 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -WhatIf
-Displays a message that describes the effect of the command instead of executing the command.
-For more information, type the following command: `get-help about_commonparameters`
+### -SMTPCredentials
+Specifies credentials to authenticate to the SMTP server.
+Set the value to $null to connect to the SMTP server anonymously.
+If this parameter isn't specified, the existing authentication settings will be preserved.
+
+You must use the \`Set-SPApplicationCredentialKey\` PowerShell cmdlet to set an application credential key on each server in the farm before specifying credentials.
```

Comment:
You may also want to mention that the App Cred Key needs to be the same value on each server in the farm.

Reply:
@TroyStarr Could you please take a look at this and give suggestions on whether we need to add something related ro App Cred Key?

Reply:
Sure, we can say something like "You must use the \`Set-SPApplicationCredentialKey\` PowerShell cmdlet to set an identical application credential key on each server in the farm before specifying credentials."